data_hub_api/docmaps/provider.py

```diff
@@ -340,7 +340,7 @@ def get_docmap_item_for_query_result_item(query_result_item: dict) -> dict:
     return {
         '@context': DOCMAPS_JSONLD_SCHEMA_URL,
         'type': 'docmap',
-        'id': DOCMAP_ID_PREFIX + query_result_item['docmap_id'],
+        'id': DOCMAP_ID_PREFIX + "'" + query_result_item['preprint_doi'] + "'",
```

Comment:
Why do we add the single quotes?
The DOIs may need to get URL encoded.

Reply:
When I tried without quotes and it didn't work :( So have added them.
encoded url may work let me try them

Reply:
I may be wrong, but typically I would think those quotes become part of the value rather than escape the value.
Maybe that is something we can look at together.
As it is no longer a simple appending of the value, it might be worth-while to put that in a separate function.